trendmicrovisionone_connector.py

```diff
@@ -0,0 +1,1770 @@
+#!/usr/bin/python
```

Comment:
Apache license is missing. Can we add it?

Reply:
Done